google-cloud-storage/lib/google/cloud/storage/file.rb

```diff
@@ -1560,6 +1562,46 @@ def delete generation: nil,
           true
         end
 
+        # Mode of object level retention configuration.
+        # Valid values are 'Locked' or 'Unlocked'
+        #
+        # @return [String]
+        def retention_mode
+          @gapi.retention&.mode
+        end
+
+        # The earliest time in RFC 3339 UTC "Zulu" format that the object can
+        # be deleted or replaced.
+        #
+        # @return [DateTime]
+        def retention_retain_until_time
+          @gapi.retention&.retain_until_time
+        end
+
+        # A collection of object level retention parameters.
+        # The full list of available options are outlined at the [JSON API docs]
+        # (https://cloud.google.com/storage/docs/json_api/v1/objects/insert#request-body).
```

Comment:
Recommend using the curly braces syntax (i.e; `{...}`) for hyperlinks in YARD. See https://rubydoc.info/gems/yard/file/docs/GettingStarted.md#linking-urls-http

Reply:
I'd generally agree but this file already has this format in multiple places. For e.g.: https://github.com/googleapis/google-cloud-ruby/pull/23732/files/213f6f9bf7a47c7bf9576aa3bcc9bd958682883b#diff-2788eaf7ebb8b45d5b11379afd02ffe326b411ad9d0841f5f0de195073c6869dR1414-R1417